web-common/src/features/dashboards/dimension-table/DimensionFilterGutter.svelte

```diff
@@ -45,14 +45,22 @@
   </div>
   {#each virtualRowItems as row (`row-${row.key}`)}
     {@const isSelected = selectedIndex.includes(row.index)}
+    {@const dimensionValue = String(rows[row.index][column.name])}
     <StickyHeader
       enableResize={false}
       position="left"
       header={{ size: config.indexWidth, start: row.start }}
     >
       <div class="py-0.5 grid place-items-center">
         {#if isSelected && !excludeMode && isBeingCompared}
-          <CheckCircle className={getColor(row.index)} size="18px" />
+          <CheckCircle
+            className="fill-{colorGetter.get(
+              $page.params.name,
```

Comment:
`metricViewName` should be available from `StateManagers`. We can probably use that to get the dashboard name and stay consistent with rest of the codebase?